### ISSUE TITLE
Clicking on grayed out fields shows weird HTML tags

### ISSUE THREAD

**Issue by floesche** [JaneliaSciComp/G4_Display_Tools#76]
If I click on a grayed out field, it allows me to go into an edit mode, but shows weird HTML tags. Only once I want to confirm the change, it tells me that this is not allowed. Is there a way to prevent the click before the user tries to enter data?

This is the case for clicking on Frame Rate, Gain, or Offset for Mode 1:

![image](https://github.com/JaneliaSciComp/G4_Display_Tools/assets/1568893/bca60c3c-0e36-4c06-b948-1df5ac275583)

**Comment by taylorlm88** [comment]
It was years ago that we set this up. At the time at least, matlab's table features were pretty limited and we decided it was not worth the amount of work it would take to fix this (if it was even possible, which we weren't sure it was without going under matlab and adding in some C code.) It's been a long time since then so I could go back and see if there are any new table functionalities that would solve this, because it would definitely be cleaner looking to not allow the user to click the cell at all. Will add this to the low priority issues to look into.
